src/doc/man/cargo-metadata.md

```diff
@@ -362,6 +362,7 @@ possible value.
 {{#option "`--filter-platform` _triple_" }}
 This filters the `resolve` output to only include dependencies for the
 given [target triple](../appendix/glossary.html#target).
+The value `host-tuple` can be used to use the host target triple.
```

Comment:
Maybe follow the existing doc? https://doc.rust-lang.org/cargo/commands/cargo-build.html#compilation-options



```suggestion
A literal `"host-tuple"` can be used, which will internally be substituted by the host’s target.
```